services/newsfeed/package.json

```diff
@@ -27,16 +27,23 @@
     "eslint": "eslint ./api ./knowledge ./services ./test ./*.js",
     "eslint-fix": "eslint --fix ./api ./knowledge ./services ./test ./*.js",
     "clean": "rm -rf ./node_modules",
-    "watch": "supervisor -w . -i ./node_modules app.js"
+    "watch": "supervisor -w . -i ./node_modules app.js",
+    "test:unit": "./node_modules/.bin/jest --config=jest.config.unit.js --detectOpenHandles --forceExit"
   },
   "dependencies": {
-    "html-entities": "^2.3.2",
-    "html-to-text": "^1.2.1",
+    "bluebird": "^3.7.2",
+    "knex": "^0.95.4",
+    "mysql": "^2.18.1",
     "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/0ba4de1b8dc37fd35f38ce122321345284f35771/framework/dist/lisk-service-framework-1.2.1.tgz",
     "moment": "^2.29.1",
+    "moleculer": "^0.14.13",
     "object-hash": "^2.2.0",
-    "stack-trace": "^1.0.0-pre1",
-    "twitter": "^1.7.1"
+    "stack-trace": "0.0.10",
```

Comment:
Please remove it (if not used) or update it to the recent version.

Reply:
Updated!